src/sol/staking_rewards.py

```diff
@@ -8,47 +8,47 @@
 from sol.make_tx import make_sol_reward_tx
 
 DATADIR = os.path.dirname(os.path.realpath(__file__)) + "/data_staking_rewards"
+START_EPOCH = 132  # epoch of first ever staking reward
 
 
 def reward_txs(wallet_info, exporter, progress):
-    """ Get reward transactions across all staking addresses for this wallet """
+    """Get reward transactions across all staking addresses for this wallet"""
     staking_addresses = wallet_info.get_staking_addresses()
     wallet_address = wallet_info.wallet_address
 
     for i, addr in enumerate(staking_addresses):
-        progress.report("staking", i, "Fetching rewards for {}...".format(addr))
+        progress.report("staking", i, f"Fetching rewards for {addr}...")
         _reward_txs(wallet_address, exporter, addr)
 
 
 def _reward_txs(wallet_address, exporter, staking_address):
-    """ Get reward transactions for this staking address """
-    start_epoch = 132  # epoch of first staking reward ever
+    """Get reward transactions for this staking address"""
     latest_epoch = RpcAPI.get_latest_epoch()
 
     rewards = []
-    for epoch in range(start_epoch, latest_epoch):
+    for epoch in range(START_EPOCH, latest_epoch):
         timestamp, reward = _get_reward(epoch, staking_address)
         if not reward:
             continue
         rewards.append([epoch, timestamp, reward])
 
     for epoch, timestamp, reward in rewards:
-        txid = "{}.{}".format(staking_address, epoch)
+        txid = f"{staking_address}.{epoch}"
         row = make_sol_reward_tx(timestamp, reward, wallet_address, txid)
         exporter.ingest_row(row)
 
 
 def _get_reward(epoch, staking_address):
-    """ Returns single reward (timestamp_of_reward, float_reward_amount) for staking_address at this epoch. """
-    flush = (localconfig.job is None)
+    """Returns single reward (timestamp_of_reward, float_reward_amount) for staking_address at this epoch."""
+    flush = localconfig.job is None
```

Comment:
is removing the parentheses a auto-linter thing?  I like with, so i don't need to think about it

Reply:
fyi black auto formatter removed it.